packages/dds/tree/src/test/feature-libraries/modularChangeFamilyIntegration.spec.ts

```diff
@@ -53,19 +53,22 @@ import {
 } from "../utils.js";
 
 import type {
-	NodeChangeset,
 	NodeId,
 	// eslint-disable-next-line import/no-internal-modules
 } from "../../feature-libraries/modular-schema/modularChangeTypes.js";
 // eslint-disable-next-line import/no-internal-modules
 import { MarkMaker } from "./sequence-field/testEdits.js";
+// eslint-disable-next-line import/no-internal-modules
+import { Change, removeAliases } from "./modular-schema/modularChangesetUtil.js";
+import { merge } from "../objMerge.js";
```

Comment:
I'm assuming this was only used for debugging and can now be removed

Reply:
Thanks. I accidentally do this a lot.

Reply:
I'm glad this function is proving useful to you. :)